src/screens/ProfileScreen/index.js

```diff
@@ -662,9 +667,7 @@ const ProfileScreen = ({route}) => {
               source={SOURCE_MY_PROFILE}
               hideThreeDot={false}
               showAnonymousOption={true}
-              onDeletePost={() => onDeletePost()}
-              isSelf={item.is_self}
-              isShowDelete={true}
+              onHeaderOptionClicked={() => onHeaderOptionClicked(item)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onHeaderOptionClicked` function is now being passed as a prop to the `ProfileHeader` component. Ensure that this prop is properly used and handled in the `ProfileHeader` component.
```diff
- onDeletePost={() => onDeletePost()}
+ onHeaderOptionClicked={() => onHeaderOptionClicked(item)}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->